src/AdvApi32.Tests/AdvApi32.cs

```diff
@@ -1,10 +1,7 @@
 ﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 
-using System;
-using PInvoke;
 using Xunit;
-using static PInvoke.AdvApi32;
```

Comment:
Removal of these using statements is a disservice, IMO. Once real tests are written, these will inevitably have to be put back. I'm guessing your tool did a 'remove unused usings' step. Can we remove that step and have the PR just extract nested types and reserve removing unused usings as a separate PR, if we can agree the result is desirable?